README.md

```diff
@@ -103,3 +103,13 @@ $ sudo apt-get install idn # Debian/Ubuntu
 $ brew install libidn # OS X
 $ gem install idn-ruby
 ```
+
+# Semantic Versioning
+
+This project conforms to semver. As a result of this policy, you can (and should) specify a dependency on this gem using the Pessimistic Version Constraint with two digits of precision. For example:
+
+spec.add_dependency 'addressable', '~> 1.0'
+
+This means your project is compatible with addressable 1.0 up until 2.0. You can also set a higher minimum version:
+
+spec.add_dependency 'addressable', '~> 1.1'
```

Comment:
This is much more flexible and more likely to be what you'd actually want to use:

`spec.add_dependency 'addressable', '~> 1.0', '>= 1.1'`